src/app/frontend/deploy/createnamespace_controller.js

```diff
@@ -67,10 +67,8 @@ export default class NamespaceDialogController {
     /** @export {!angular.FormController} */
     this.namespaceForm;
 
-    /**
-     * @export
-     */
-    this.i18n = i18n;
+    /** @export */
+    this.maxLength = 64;
```

Comment:
previous message said 63, while indeed the spec says it is 64, why change it now hidden in this huge refactor CL?